<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PK AVR Calculator     ver. Alpha</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>491</width>
      <height>241</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Programator AVR</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>391</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:  #dcdbff;</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="ERR_Main_Label">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>20</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:  #dcdbff;
color: blue;
font-weight: bold;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="Main_button">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Sprawdź podłączony AVR</string>
      </property>
     </widget>
     <widget class="QComboBox" name="SCK_list">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>10</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maxVisibleItems">
       <number>13</number>
      </property>
     </widget>
     <widget class="QComboBox" name="Prog_list">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Pomoc</string>
     </attribute>
     <widget class="QPushButton" name="LPT_Prog_Schem_Butt">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Najprostszy programator LPT</string>
      </property>
     </widget>
     <widget class="QPushButton" name="KANDA_help">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Standard KANDA -  Bez Problemu</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ATtiny_CON">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Podłączenie ISP ATtiny 4/5/9/10</string>
      </property>
     </widget>
     <widget class="QPushButton" name="UART_CALC">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>UART/USART Calculator</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QComboBox" name="uC_list">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maxVisibleItems">
     <number>20</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>516</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
